.circleci/config.yml

```diff
@@ -12,9 +12,9 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "requirements-test.txt" }}
+          - v2-dependencies-{{ checksum "requirements-test.txt" }}
```

Comment:
Why the change from `v1` -> `v2`?

Reply:
The circleci pipeline was failing, because to the cache contained outdated packages. I modified the key prefix as recommended in the [circle-ci documentation](https://support.circleci.com/hc/en-us/articles/115015426888-Clear-project-cache).
However, there must be a better solution... (the same pipeline fails now). I don't know enough about CircleCI to fix this...

Reply:
I think we will switch to Travis as most OT projects use that one ;)